### PR TITLE
grid used

### PR DESCRIPTION
**Pull request by pr1syazhniy**
[DEMO LINK](https://pr1syazhniy.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -19,4 +19,16 @@
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, auto);
```
Comment (vlad-demchuk) [inline src/styles/_catalog.scss]: width shouldn't be auto, specify it

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -17,35 +17,35 @@
...
-      <a class="header__logo" href="">
+      <a class="header__logo" href="@">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="@" class="nav__link">Apple</a>
```
Comment (anstsot) [inline src/index.html]: add the proper links in all cases
```suggestion
            <a href="#apple" class="nav__link">Apple</a>
```

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,9 +1,9 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```
Comment (anstsot) [inline src/styles/_catalog.scss]: please specify the number of cards for all media queries. now it seems to be broken: 
image
image

**Review by anstsot — CHANGES_REQUESTED** [review]
Good work, but the conditions of the task were not met

**Review by nazarmatsevych — APPROVED** [review]
👍

**Pull request closed without merging**